test/ginkgo-ext/scopes.go

```diff
@@ -257,7 +258,7 @@ func runAllAfterFail(cs *scope, testName string) {
 		for _, body := range cs.afterFail {
 			body()
 		}
-		GinkgoPrint("===================== Existing AfterFailed =====================")
```

Comment:
I liked this one ;)

Reply:
I almost wanted to keep it for posterity:)